content/scripts/check-pr-links.rb

```diff
@@ -122,8 +142,7 @@ def find_links(html)
   puts '=== No broken links! ==='
 else
   puts "=== Found broken links! ==="
-  puts "Fix before merging... or if they're not really broken, explain why."
-  puts "(NOTE: This script reports false positives for Vercel app routes like /cloud. Sorry!)\n\n"
+  puts "Fix before merging... or if they're not really broken, explain why.\n\n"
```

Comment:
```suggestion
  puts 'Fix before merging... or if they're not really broken, explain why.\n\n'
```
No interpolation so no need for double quotes

Reply:
Alas, the `\n` newlines require doubles.